funtimes/R/BICC.R

```diff
@@ -0,0 +1,40 @@
+BICC <- function(x, N = NULL, WINDOW = NULL, p = NULL, SHIFT = NULL, TREND = NULL)
+  
+{
+  for (window in 1:length(WINDOW)){
+    for (shift in 1:length(SHIFT)){
+      s <- window  #step to shift a window
+      #---------------------------------------------------------------------------------------------
+      #Find IQR of Data(x) and Corresponding DELTA and the residual
+      
+      IQRx <- median(apply(x,2,IQR))
+      DELTA <- seq(IQRx/100, IQRx, IQRx/N)
+      EPS <- seq(0.0, 1.0, 1.0/WINDOW[window])
+      
+      #---------------------------------------------------------------------------------------------
+      #Apply Clustering to Data(x) using the DELTA values above in order to obtain BIC
+      library(funtimes)
```

Comment:
`library() ` should not be used in a function. Assume that the package is already loaded, so all functions from this package are available.